src/sidebar/store/modules/groups.ts

```diff
@@ -12,7 +12,7 @@ type GroupID = Group['id'];
  * 'loading': Members being currently loaded
  * GroupMember[]: Members already loaded
  */
-type FocusedGroupMembers = null | 'loading' | GroupMember[];
+export type FocusedGroupMembers = null | 'loading' | GroupMember[];
```

Comment:
This existing type has the same issue that was just resolved in this PR. It would be good to refactor it along the same lines in future.

Reply:
Yep.